mon-entreprise/source/pages/Simulateurs/Dividendes.tsx

```diff
@@ -0,0 +1,104 @@
+import { useContext } from 'react'
+import { SimulationConfig } from 'Reducers/rootReducer'
+import { Condition } from 'Components/EngineValue'
+import { SimulationGoals, SimulationGoal } from 'Components/SimulationGoals'
+import { ThemeColorsContext } from 'Components/utils/colors'
+import useSimulationConfig from 'Components/utils/useSimulationConfig'
+import SimulateurWarning from 'Components/SimulateurWarning'
+import Notifications from 'Components/Notifications'
+import Simulation from 'Components/Simulation'
+import { Trans, useTranslation } from 'react-i18next'
+import StackedBarChart from 'Components/StackedBarChart'
+
+export default function DividendesSimulation() {
+	return (
+		<>
+			{/* [XXX] Seems useless */}
+			<SimulateurWarning simulateur="dividendes" />
```

Comment:
(Pas lié à cette ligne mais c'est pour faire un thread)

Si le simulateur s’adresse aux indépendants qui ont une société unipersonnelle à l'IS, je pense que la question qu'ils se posent est : « combien ça va me coûter de sortir 10000€ en dividende de mon entreprise », et que pour répondre à ça il faut aussi intégrer le calcul de l'impôt sur les sociétés.

C'est un peu contradictoire avec ma remarque sur les réserves, car par définition l'IS à déjà été payé sur les montants mis en “report à nouveau”, et que ça serait impossible de le ré-intégrer dans le calcul (il faudrait connaître tout l'historique des bénéfices).

Du coup je ne sais pas trop comment exposer ce point dans l'interface. Peut-être juste un avertissement qui prévient qu'il faut payer l'IS (avec un lien vers le simulateur dédié) avant de pouvoir se verser un dividende. On pourrait aussi imaginer proposer les deux modes (avec et sans IS) mais ça me paraît compliqué dans une première version.

Reply:
Pour l'instant faisons simplement un message. Je vais réfléchir rapidement à à quoi pourrait ressembler un simulateur où tu fois et l'IS bénéfices et l'IR dividendes, mais vu d'ici j'aurais tendance à le voir en mode "meta-simulateur" dans le dashboard qu'on veut construire.

Reply:
On peut imaginer une vue "flux" aussi

![IMG_20210706_154148_edit_81663509047954.jpg](https://user-images.githubusercontent.com/1730702/124610625-f7cc5900-de70-11eb-80ae-aa4ebb5ed6db.jpg)